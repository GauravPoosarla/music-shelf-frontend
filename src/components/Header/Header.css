* {
  padding: 0;
  margin: 0;
}

.header {
  height: 50px;
  background-color: #38497E ;
  color: white;
  padding: 5px 0px 0px 15px;
  font-size: 30px;
}

@media screen and (max-width: 768px) {
  .header {
    font-size: 25px;
    
  }
}
