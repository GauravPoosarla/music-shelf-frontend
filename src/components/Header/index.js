import React from 'react';
import './Header.css';

function NavBar() {
  return (
    <div className="header">
      <p>
        My <strong> Record </strong> Shelf
      </p>
    </div>
  );
}

export default NavBar;
