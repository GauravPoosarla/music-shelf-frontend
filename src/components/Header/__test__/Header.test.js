import React from 'react';
import { render, screen, fireEvent, waitFor } from '@testing-library/react';

import Header from '..';

describe('Header', () => {
  it('should render the header', () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
