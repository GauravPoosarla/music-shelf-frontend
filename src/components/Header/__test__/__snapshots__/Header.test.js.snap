// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render the header 1`] = `
<div>
  <div
    class="header"
  >
    <p>
      My 
      <strong>
         Record 
      </strong>
       Shelf
    </p>
  </div>
</div>
`;
